packages/integrations/test/extraerrordata.test.ts

```diff
@@ -178,4 +178,42 @@ describe('ExtraErrorData()', () => {
       },
     });
   });
+
+  it('captures Error causes when captureErrorCause = true', () => {
+    const extraErrorDataWithCauseCapture = new ExtraErrorData({ captureErrorCause: true });
+
+    // @ts-expect-error TS apparently doesn't understand error causes
```

Comment:
Can we add a comment to here?

> If we examine [the core TS library declaration files](https://github.com/microsoft/TypeScript/tree/v4.7.4/lib) for ES features (in v4.7.4), we find that the types related to the error cause property are located in [lib.es2022.error.d.ts](https://github.com/microsoft/TypeScript/blob/v4.7.4/lib/lib.es2022.error.d.ts). These types are used in [lib.es2022.d.ts](https://github.com/microsoft/TypeScript/blob/v4.7.4/lib/lib.es2022.d.ts) and later versions, but are actually not used in [lib.es2021.d.ts](https://github.com/microsoft/TypeScript/blob/v4.7.4/lib/lib.es2021.d.ts).

Therefore, we need to set the lib option to at least "es2022".

Reply:
We cannot set the lib option to "es2022" because that would be wrong for our lower support bound but I can definitely add a comment.